scipy/stats/_distr_params.py

```diff
@@ -284,7 +284,7 @@
     ['triang', (2, )],
     ['truncexpon', (-1, )],
     ['truncnorm', (10, 5)],
-    ['truncpareto', (-1, 5)],
+    ['truncpareto', (-1, -5)],
```

Comment:
One of the failures was a test that checks that all distributions return `nan`s when the distribution parameters are invalid. This parameter combination had become valid, so this makes it invalid again.